src/lib/localization.js

```diff
@@ -22,6 +22,8 @@ function NumberFormatter(locale, options) {
         locale = undefined;
     }
 
```

Comment:
No, as different cells (or columns) can have different locals — in particular for showing different currencies.